src/bankid.ts

```diff
@@ -361,10 +370,42 @@ export class BankIdClient {
     const passphrase = this.options.passphrase;
 
     return axios.create({
+      baseURL,
       httpsAgent: new https.Agent({ pfx, passphrase, ca }),
       headers: {
         "Content-Type": "application/json",
       },
     });
   }
 }
+
+interface AuthOptionalRequirementsV6 {
+  pinCode: boolean;
+  cardReader?: "class1" | "class2";
+  mrtd: boolean;
+  certificatePolicies?: string[];
+  personalNumber: string;
+}
+
+export interface AuthRequestV6 {
+  endUserIp: string;
+  requirement?: AuthOptionalRequirementsV6;
+}
+
+interface AuthResponseV6 extends AuthResponse {
+  qr: InstanceType<typeof QrGenerator>;
```

Comment:
Isn't this sufficient?

```suggestion
  qr: QrGenerator;
```